numpy/f2py/tests/test_crackfortran.py

```diff
@@ -33,3 +35,50 @@ def test_module(self):
         self.module.subc([w, k])
         assert_array_equal(k, w + 1)
         assert self.module.t0(23) == b'2'
+
+class TestPublicPrivate():
+    def test_defaultPrivate(self):
+        with tempfile.NamedTemporaryFile() as ff:
+            ff.write(b"""
+module foo
+  private
+  integer :: a
+  public :: setA
+contains
+  subroutine setA(v)
+    integer, intent(in) :: v
+    a = v
+  end subroutine setA
+end module foo
+""")
```

Comment:
```suggestion
            ff.write(textwrap.dedent(b"""\
            module foo
              private
              integer :: a
              public :: setA
            contains
              subroutine setA(v)
                integer, intent(in) :: v
                a = v
              end subroutine setA
            end module foo
            """))
```

Reply:
I've modified the source to use the textwrap dedent() function.